main.py

```diff
@@ -16,6 +16,7 @@
 API.add_resource(REST.release, '/release')
 API.add_resource(REST.labels, '/labels')
 API.add_resource(REST.tasks, '/tasks')
+API.add_resource(REST.airflowdb, '/airflowdb')
```

Comment:
Why do we need a API for this ?

Reply:
This will eventually be removed, it's there right now for my own personal testing

Reply:
Done